README.md

```diff
@@ -49,6 +49,12 @@ For a graphical user interface in a desktop application, see the [napari plugin]
 Acknowledgements
 ----------------
 
+ITKElastix was developed in part by with support from:
+
+- [NIH NIMH BRAIN Initiative](https://braininitiative.nih.gov/) under award 1RF1MH126732.
+- Chen Zuckerberg Initiative (CZI) Essential Open Source Software for Science award for [Open Source Image Registration: The elastix
```

Comment:
Thanks Matt! Just a typo here: her name is Chan, not Chen!

Reply:
Good catch, Neils! :eagle: :eye: